test/test_treetensornetworks/test_solvers/test_tdvp.jl

```diff
@@ -24,14 +25,14 @@ using Test
     ψ0 = random_mps(s; internal_inds_space=10)
 
     # Time evolve forward:
-    ψ1 = tdvp(H, -0.1im, ψ0; nsteps=1, cutoff, nsite=1)
+    ψ1 = tdvp(H, -0.1im, ψ0; nsteps=1, cutoff, nsites=1)
 
     #
-    #TODO: exponentiate is now the default, so switch this to applyexp
+    #TODO: applyexp is removed, no need to test a different backend here and below for now.
     #
-    #Different backend solvers, default solver_backend = "applyexp"
+    #Different backend updaters, default updater_backend = "applyexp"
```

Comment:
Seems like we should just remove this comment.

Reply:
Or is this a reminder to delete some of these tests, perhaps in a future PR?

Reply:
It seems to me that it is not crucial to test whether passing a user-defined solver works, since the whole code design relies on this being possible (is indirectly also tested in the time-dependent tests.). So can just remove these tests and comments now?

Reply:
Yeah that would be better.

Reply:
I removed the related tests.